src/picongpu/submit/bash/bash_mpiexec.tpl

```diff
@@ -43,9 +43,17 @@ umask 0027
 mkdir simOutput 2> /dev/null
 cd simOutput
 
-mpiexec --prefix $MPI_ROOT -tag-output --display-map -x LIBRARY_PATH -x LD_LIBRARY_PATH -am !TBG_dstPath/tbg/openib.conf --mca mpi_leave_pinned 0 -npernode !TBG_gpusPerNode -n !TBG_tasks !TBG_dstPath/picongpu/bin/cuda_memtest.sh
+# test if cuda_memtest binary is available
+test -f !TBG_dstPath/picongpu/bin/cuda_memtest
+haveCudaMemtest=$?
+
+if [ $haveCudaMemtest -eq 0 ] ; then
```

Comment:
since the line below is not necessary, you could do `if [ -f  ...]` here directly